setup.py

```diff
@@ -0,0 +1,31 @@
+import pathlib
```

Comment:
is importing just the module a stylistic choice or something else? otherwise why not:

```python
from pathlib import Path
from shutil import rmtree
from shutil import copytree
```
just curious

Reply:
Just stylistic, I don't feel strongly about this.